### PR TITLE
task/20_GraphQL_Basic: Finished summary and essay assignment

### PR DESCRIPTION
**Pull request by daffadmr**
- [x] Make summary in README.md  
- [x] Add query and mutation in .json file to praktikum folder
- [x] Add screenshot hasil praktikum

### CODE REVIEW

File: 20_GraphQL_Basic/praktikum/getTrueStatus.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+query getTrueStatus {
+  kampus_merdeka_keterangan(where: {status: {_eq: true}}) {
+    id
+    id_anggota
+    nilai
+    pelajaran
+    status
+  }
+}
```
Comment (syukronarie) [inline 20_GraphQL_Basic/praktikum/getTrueStatus.json]: ini seharusnya format filenya .graphql

File: 20_GraphQL_Basic/praktikum/insertNilaiAnggaToKeterangan.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  insert_kampus_merdeka_keterangan(objects: {id_anggota: $id_anggota, nilai: $nilai, pelajaran: $pelajaran, status: $status}) {
+    returning {
+      id
+      id_anggota
+      nilai
+      pelajaran
+      status
+    }
+  }
+}
```
Comment (syukronarie) [inline 20_GraphQL_Basic/praktikum/insertNilaiAnggaToKeterangan.json]: ini seharusnya format filenya .graphql

**Review by syukronarie — APPROVED** [review]

File: 20_GraphQL_Basic/praktikum/insertNilaiAnggaToKeterangan.json (hunk 2 above)
Comment (daffadmr) [reply 20_GraphQL_Basic/praktikum/insertNilaiAnggaToKeterangan.json]: Sudah saya ubah mas format filenya

File: 20_GraphQL_Basic/praktikum/getTrueStatus.json (hunk 1 above)
Comment (daffadmr) [reply 20_GraphQL_Basic/praktikum/getTrueStatus.json]: Sudah saya ubah mas format filenya

**Pull request merged**